### PR TITLE
Waypoint Plugin Integrations Docs: Part 2

### PR DESCRIPTION
**Pull request by paladin-devops**
This PR contains the plugin integrations docs for the remaining plugins.

### CODE REVIEW

File: builtin/docker/ref/metadata.hcl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 integration {
   name        = "Docker Ref"
-  description = "TODO"
+  description = "The Docker Ref plugin refers to an existing Docker image, passing its image information, the image name and tag, to the Waypoint lifecycle."
```
Comment (krantzinator) [inline builtin/docker/ref/metadata.hcl]: nit
```suggestion
  description = "The Docker Ref plugin refers to an existing Docker image, passing its image information - the image name and tag - to the Waypoint lifecycle."
```

File: builtin/packer/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Packer
+
+The Packer plugin retrieves the image ID of an image whose metadata is pushed 
+to an [HCP Packer](https://cloud.hashicorp.com/products/packer) registry. The
```
Comment (krantzinator) [inline builtin/packer/README.md:4]: 🏆

**Review by krantzinator — APPROVED** [review]
I am so excited 🤩

**Review by BrandonRomano — COMMENTED** [review]
@krantzinator LMK if you'd like me to merge this and I can :)

**Pull request merged**